java/test_flatbuf1/test.dump.expected

```diff
@@ -15,24 +15,24 @@ From 13 Insert HO{773153186440202938}
 From 15 Insert IO{7}
 From 17 Insert JO{(true, 10, "string")}
 From 19 Insert KO{(false, 9, "text")}
-From 21 Insert L0O{false,254,"else"}
-From 23 Insert (true, 255, "something")
+From 21 Insert L0O{false, -2, "else"}
```

Comment:
L0O has a `bit<8>` value in the second field. 
Do you expect it to be printed as a negative number?
If this is the intended behavior we should document it. In particular we have to document how each DDlog type is mapped to a Java type.